API.md

```diff
@@ -262,6 +262,50 @@ export class URLSearchParams {
 
 [TextEncoder](https://nodejs.org/api/util.html#class-utiltextdecoder)
 
+## zlib
+
+[deflate](https://nodejs.org/api/zlib.html#zlibdeflatebuffer-options-callback)
+
+[deflateSync](https://nodejs.org/api/zlib.html#zlibdeflatesyncbuffer-options)
+
+[deflateRaw](https://nodejs.org/api/zlib.html#zlibdeflaterawbuffer-options-callback)
+
+[deflateRawSync](https://nodejs.org/api/zlib.html#zlibdeflaterawsyncbuffer-options)
+
+[gzip](https://nodejs.org/api/zlib.html#zlibgzipbuffer-options-callback)
+
+[gzipSync](https://nodejs.org/api/zlib.html#zlibgzipsyncbuffer-options)
+
+[inflate](https://nodejs.org/api/zlib.html#zlibinflatebuffer-options-callback)
+
+[inflateSync](https://nodejs.org/api/zlib.html#zlibinflatesyncbuffer-options)
+
+[inflateRaw](https://nodejs.org/api/zlib.html#zlibinflaterawbuffer-options-callback)
+
+[inflateRawSync](https://nodejs.org/api/zlib.html#zlibinflaterawsyncbuffer-options)
+
+[gunzip](https://nodejs.org/api/zlib.html#zlibgunzipbuffer-options-callback)
+
+[gunzipSync](https://nodejs.org/api/zlib.html#zlibgunzipsyncbuffer-options)
+
+[brotliCompress](https://nodejs.org/api/zlib.html#zlibbrotlicompressbuffer-options-callback)
+
+[brotliCompressSync](https://nodejs.org/api/zlib.html#zlibbrotlicompresssyncbuffer-options)
+
+[brotliDecompress](https://nodejs.org/api/zlib.html#zlibbrotlidecompressbuffer-options-callback)
+
+[brotliDecompressSync](https://nodejs.org/api/zlib.html#zlibbrotlidecompresssyncbuffer-options)
+
+
+> [!NOTE]
+> The following functions are LLRT's own implementation of Zstandard format compression/decompression.
+
+- zstandardCompress
```

Comment:
While these are nice to have I'm a bit unsure wether we should add them to the runtime. We have historically tried to stayed clear of custom LLRT APIs (expect for some Lambda specific implementations) as these are not existent in Node.

Reply:
Thats why I was suggesting we tag the APIs, I imagine some people will want to add WinterGC APIs that are not Node based too at some point.
Anyway this zlib is a good step but we will really need a stream based API for anything serious (that you cant decompress all in RAM).

Reply:
@richarddavison and @Sytten , I apologize for the confusion caused by my unnecessary implementation. I will remove the Zstandard implementation until the policy is clarified.

If this becomes necessary before the policy is clarified, I believe it can be provided from the llrt namespace.